t.py

```
@@ -1,54 +1,38 @@
+import os
 from pprint import pprint
```

Ruff

`pprint.pprint` imported but unused

Remove unused import: `pprint.pprint`

(F401)